<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSheetForm</class>
 <widget class="QWidget" name="DataSheetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据查询</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_main">
       <item>
        <widget class="QToolButton" name="toolButton_alarm">
         <property name="font">
          <font>
           <family>等线</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>运行记录</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_info">
         <property name="font">
          <font>
           <family>等线</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>报警记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_operate">
         <property name="font">
          <font>
           <family>等线</family>
           <pointsize>12</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>操作记录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_view">
       <item>
        <widget class="QTableView" name="tableView_Sheet"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pageChange" stretch="2,0,1,1,0,0,0,2">
         <item>
          <widget class="QPushButton" name="pushButton_previousPage">
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_firstPage">
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_currentPage">
           <property name="text">
            <string>第几页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pageCount">
           <property name="text">
            <string>共几页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_gotoPage">
           <property name="text">
            <string>跳转</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_gotoPage"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_lastPage">
           <property name="text">
            <string>末页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nextPage">
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_condition">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_address"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_address">
              <property name="text">
               <string>设备地址:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>结束时间:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateTimeEdit" name="dateTimeEdit_start">
              <property name="displayFormat">
               <string>yyyy-MM-dd HH:mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateTimeEdit" name="dateTimeEdit_end">
              <property name="displayFormat">
               <string>yyyy-MM-dd HH:mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>起始时间:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_button">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_query">
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_output">
              <property name="text">
               <string>导出</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_print">
              <property name="text">
               <string>打印</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_pdf">
              <property name="text">
               <string>PDF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_status">
            <item>
             <widget class="QLineEdit" name="lineEdit_timeCosume">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_documentCount">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_CountPerPage">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
